components/courses/app/controllers/admin/courses/seasons_controller.rb

```diff
@@ -3,8 +3,10 @@ module Courses
     class SeasonsController < BaseController
       helper_method :seasons, :season
       add_breadcrumb 'seasons.plural', :admin_courses_seasons_path
+      before_action :add_season_breadcrumb, only: %i[show edit update]
       before_action :add_new_breadcump,  only: %i[new create]
       before_action :add_edit_breadcump, only: %i[edit update]
+      before_action :authenticate_mentor, only: %i[show edit update]
```

Comment:
Here is ok, but you can use except:
`before_action :authenticate_mentor, except: :index`